packages/story-editor/src/components/library/panes/media/local/mediaRecording/imageCapture.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { __ } from '@googleforcreators/i18n';
+import { useRef } from '@googleforcreators/react';
+import styled from 'styled-components';
+import { blobToFile, getImageFromVideo } from '@googleforcreators/media';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+} from '@googleforcreators/design-system';
+
+const Photo = styled.img`
+  display: none;
+  width: 240px;
+  margin: 0 auto;
+`;
+
+const Actions = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
+`;
+
+/**
+ * Returns a Base64 image from a given video element.
+ *
+ * @param {blob} blob from Video element.
+ * @return {Promise<string>} Base64 image.
+ */
+function blobToBase64(blob) {
+  return new Promise((resolve) => {
+    const reader = new FileReader();
+    reader.onloadend = () => resolve(reader.result);
+    reader.readAsDataURL(blob);
+  });
+}
+
+export const ImageCapture = ({ videoRef, onCapture, onClear }) => {
+  const photoRef = useRef();
+
+  const clear = () => {
+    photoRef.current.setAttribute('src', '');
+    photoRef.current.style.display = 'none';
+    onClear();
+  };
+
+  const captureImage = async () => {
+    if (!videoRef.current || !photoRef.current) {
+      return;
+    }
+    const blob = await getImageFromVideo(videoRef.current);
+    photoRef.current.setAttribute('src', await blobToBase64(blob));
+    photoRef.current.style.display = 'block';
+    onCapture(blobToFile(blob, 'image-capture.png', 'image/png'));
```

Comment:
We have used jpeg for other poster images. Let's re-use `MEDIA_POSTER_IMAGE_MIME_TYPE` here.